druntime/test/shared/src/finalize.d

```diff
@@ -44,7 +44,7 @@ void main(string[] args)
     auto nf1 = new NoFinalize;
     auto nf2 = new NoFinalizeBig;
 
-    shared size_t finalizeCounter;
+    static shared size_t finalizeCounter;
```

Comment:
Must be a global (not on the stack), as due to the dlclose no-op, the finalizers might run after main() ends, causing a segfault.